authorize/evaluator/opa/policy/authz.rego

```diff
@@ -166,7 +169,9 @@ jwt_payload_jti = v {
 }
 
 jwt_payload_exp = v {
-	v = session.expires_at.seconds
+	v = min([five_minutes, session.id_token.expires_at.seconds])
```

Comment:
could we add a test for the case where the session expiry is less than five minutes?